src/main/java/pages/LoginPage.java

```diff
@@ -113,5 +126,101 @@ public String getSignInButtonFontColorAsHex() {
+    }
+
+    public boolean isPageNavigationBarDisplayed(){
+        return driver.findElement(pageNavigationBar).isDisplayed();
+    }
+
+    public boolean isPageFooterDisplayed(){
+        return driver.findElement(pageFooter).isDisplayed();
+    }
+
+    public boolean isNewCustomerContainerDisplayed(){
+        return driver.findElement(newCustomerContainer).isDisplayed();
+    }
+
+    public boolean isTryDemoCustomerAccessSectionDisplayed(){
+        return driver.findElement(tryDemoCustomerAccessSection).isDisplayed();
+    }
+
+    public boolean isNewCustomersTitleDisplayed(){
+        return driver.findElement(newCustomersTitle).isDisplayed();
+    }
+
+    public String getNewCustomersTitleText() {
+        return driver.findElement(newCustomersTitle).getText();
+    }
+
+    public boolean isNewCustomersMainTextDisplayed(){
+        return driver.findElement(newCustomersMainText).isDisplayed();
+    }
+
+    public String getNewCustomersMainText() {
+        return driver.findElement(newCustomersMainText).getText();
+    }
+
+    public boolean isCreateAccountButtonEnabled(){
+        return driver.findElement(createAccountButton).isDisplayed();
+    }
+
+    public String getCreateAccountButtonText() {
+        return driver.findElement(createAccountButton).getText();
+    }
+
+    public boolean isTryDemoCustomerAccessTitleDisplayed(){
+        return driver.findElement(tryDemoCustomerAccessTitle).isDisplayed();
+    }
+
+    public String getTryDemoCustomerAccessTitleText() {
+        return driver.findElement(tryDemoCustomerAccessTitle).getText();
+    }
+
+    public boolean isTryDemoCustomerAccessEmailDisplayed(){
+        return driver.findElement(tryDemoCustomerAccessEmail).isDisplayed();
+    }
+
+    public String getTryDemoCustomerAccessEmailText() {
+        return driver.findElement(tryDemoCustomerAccessEmail).getText();
+    }
+
+    public boolean isTryDemoCustomerAccessPasswordDisplayed(){
+        return driver.findElement(tryDemoCustomerAccessPassword).isDisplayed();
+    }
+
+    public String getTryDemoCustomerAccessPasswordText(){
+        return driver.findElement(tryDemoCustomerAccessPassword).getText();
+    }
+
+    public String getCreateAccountButtonFontColorAsHex() {
+        WebElement createAccountButtonElement = driver.findElement(createAccountButton);
+        String createAccountButtonColorFontRgba = createAccountButtonElement.getCssValue("color");
+        return Color.fromString(createAccountButtonColorFontRgba).asHex();
+    }
+
+    public String getCreateAccountButtonBackgroundColorAsHex(){
+        WebElement createAccountButtonElement = driver.findElement(createAccountButton);
+        waitUntilElementContainsText(driver,createAccountButton,"Create an Account");
+        String createAccountButtonColorBackgroundRgba = createAccountButtonElement.getCssValue("background");
+        return Color.fromString(createAccountButtonColorBackgroundRgba).asHex();
```

Comment:
The test will failed on this part, as the css value stored in createAccountButtonColorBackgroundRgba is not only rgb, but `rgb(25, 121, 195) none repeat scroll 0% 0% / auto padding-box border-box`. As the Color.fromString() method is not able to parse the whole css value, as it expects only rgb value.

There are 2 approach to resolve the issue. You can choose of them or implement your own.
- create a helper method that accepts string containing css value for a web element and then return substring containing only rgb value
- not trying to covert getting cssValue to hex and use expected rgb value with  `contains()` method instead of `equalTo()` assertion method in the test.

Note: if you are not able to identify error reason or you are not sure about the value stored in the given variable, you can always why writing test use `System.out.println()` method. It really helps while debugging issues.